app-samples/Face/Cognitive-Services-Face-WPF/Sample-WPF/packages.config

```diff
@@ -0,0 +1,10 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Microsoft.Azure.CognitiveServices.Vision.Face" version="2.3.0-preview" targetFramework="net452" />
```

Comment:
also change this if you change Framework

Reply:
Upgraded to `4.6`.